keystore/filesystem/server_keystore_test.go

```diff
@@ -495,17 +495,17 @@ func TestFilesystemKeyStoreExport(t *testing.T) {
 	}
 	// Since we cannot access all generated key pairs via AcraServer keystore,
 	// we generate them here and use Save... API
-	connectorKeyPair, err := keys.New(keys.TypeEC)
+	connectorKeyPair, _ := keys.New(keys.TypeEC)
```

Comment:
it's anti-pattern to avoid error check in golang. even if you know that this code always returns `nil` inside, you should check error value, because in a future somebody can change it and expect that it checked in other pieces of code. even if it tested above or below. In such cases you can create adapter which always will check error and encapsulate it for other callers:
```
func generateKeyPair(t *testing.T) *keys.KeyPair {
  keypair, err := keys.New(keys.TypeEC)
  if err != nil {t.Fatal(err)}
  return keypair
}

func TestSomething (t *testing.T){
  connectorKeyPair := generateKeyPair(t)
}
```

Reply:
agree